tests/test_cli.py

```diff
@@ -1,12 +1,24 @@
 from click.testing import CliRunner
 
 from cli import cli, consent
+from functools import wraps
 
 
 def skip_consent(monkeypatch):
     monkeypatch.setattr('cli.consent.check_consent', lambda: True)
 
 
+def with_default_env(fn):
```

Comment:
Maybe just add a note describing that this ensures the current actual CLI config isn't overwritten.